README.md

```diff
@@ -97,11 +103,154 @@ Please see Hazelcast Node.js [code samples](https://github.com/hazelcast/hazelca
 
 You can also refer to Hazelcast Node.js [API Documentation](http://hazelcast.github.io/hazelcast-nodejs-client/api/current/docs/).
 
-# Configuration
+# Configuration Overview
```

Comment:
Empty titles are not good to have. I think we can get rid of `Configuration Options` title and tell what it says there just under this title

Reply:
I added a description here so the title is not empty.